Makefile

```diff
@@ -0,0 +1,52 @@
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es3.json --env es3
```

Comment:
No, not in the browser. We rely on terminal colors, `process.stdout` and possibly more Node related things. I haven't included any Bower or UMD stuff either.